string trip_id
GpsCoordinat[] shapes
StopType[] stops
int32 zone_id
int32 vehicle_id
string service_station_id
